app/lib/features/zync_config.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Features
+  class ZyncConfig < Config
+    def initialize(...)
+      super(...)
+      config[:sync_non_oidc_apps] ||= true
```

Comment:
This is for backwards compatibility... In case any on-premises user has changed `features.yml` manually that obviously doesn't include `zync:` key, their non-OIDC apps synchronization would stop working otherwise.

But it's arguable whether it's really needed or not:

1. I think it's very unlikely somebody has changed that `features.yml` in their setup
2. It is probably even more unlikely that somebody is really willing to receive updates about non-OIDC apps in Zync (in fact we considered making this behavior default and non-configurable)

So, maybe we can just drop this customization and just add a note to the release notes.

Reply:
yeah, a default of false makes sense to me, as we spoke. But do as you find more appropriate.